## To cite ChIP-Atlas in your publication:

- Preprint
  - Shinya Oki, Tazro Ohta, et al. Integrative analysis of transcription factor occupancy at enhancers and disease risk loci in noncoding genomic regions. bioRxiv 262899; doi: https://doi.org/10.1101/262899
- Database
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://dx.doi.org/10.18908/lsdba.nbdc01558-000
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://chip-atlas.org

---

## Publications citing ChIP-Atlas:

1. C. Cui et al., Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics, Proteomics Bioinforma__. 16, 200–211 (2018), [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
2. J. B. Studd et al., Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__. In press, 1–14 (2018), [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
3. H. Kawaji, dirHub: a trackHub configurator with directory structure projection. __bioRxiv__, 314807 (2018), [Link](https://doi.org/doi:10.1101/314807).
4. A. Misawa, H. Orimo, lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif. Tissue Int.__, 1–9 (2018), [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
5. T. Kehl et al., REGGAE: a novel approach for the identification of key transcriptional regulators. __Bioinformatics__ (2018), [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
6. L. M. Raffield et al., Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Gene
7. K. Anan et al., LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res.__ (2018), [Link](https://doi.org/doi:10.1093/nar/gky234).
8. C. A. Lareau et al., Interrogation of human hematopoiesis at single-cell and single-variant resolution. __bioRxiv__, 255224 (2018), [Link](https://doi.org/doi:10.1101/255224).
9. E. Ferris, L. M. Abegglen, J. D. Schiffman, C. Gregg, Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Tr
10. S. Chatterjee et al., Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res. Treat__. 170, 431–438 (2018), [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
11. I. Miura et al., DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J. Psychiatr. Res__. 100, 84–87 (2018), [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
12. R. Mourad, K. Ginalski, G. Legube, O. Cuvier, Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol.
13. F. Mao et al., EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res__. 46, D92–D99 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx918).
14. Y. Nishizawa et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__. 9 
15. T. Chishima, J. Iwakiri, M. Hamada, Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes (Basel)__. 9, 23 (2018), [Link](https://doi.org/doi:10.3390/genes9010023).
16. J. Chèneby, M. Gheorghe, M. Artufel, A. Mathelier, B. Ballester, ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res__. 46, D267–D275 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx1092).
17. R. Dréos, G. Ambrosini, R. Groux, R. C. Périer, P. Bucher, MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res__. 46, D175–D180 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx995).
18. S. L. Romero-Cordoba et al., Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci. Rep__. 8, 12252 (2018), [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
19. P. Fiziev, J. Ernst, ChromTime: modeling spatio-temporal dynamics of chromatin marks. __Genome Biol__. 19, 109 (2018), [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
20. Ö. Åkerborg et al., High-resolution regulatory maps connect cardiovascular risk variants to disease related pathways. __bioRxiv__, 376699 (2018), [Link](https://doi.org/doi:10.1101/376699).
21. H. Imrichova, S. Aerts, ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__, 388934 (2018), [Link](https://doi.org/doi:10.1101/388934).
22. H. Wang et al., J. Biol. Chem., in press, [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
23. Z. Kalender Atak et al., Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __24. Y. Onodera et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__. 16, 1369–1380 (2017), [Link](https://doi.org/doi:10.1111/acel.12680).
25. T. Umeyama, T. Ito, DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep__. 21, 289–300 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
26. A. Naderi, C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__. 
27. Y. Chen, M. Widschwendter, A. E. Teschendorff, Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inact
28. T. Sanosaka et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep__. 20, 2992–3003 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
29. O. Govaere et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __30. M. Yoshihara et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep__. 21, 308–315 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
31. F. K. Turrell et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sens
32. M. H. Guo et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc. Natl. Acad. Sci__. 114, E327–E336 (2017), [Link](https://doi.org/doi:10.1073/pnas.1619052114).
33. T. Kehl et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res__. 45, W146–W153 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx350).
34. K. Matsuda et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__. 144, 1948–1958 (2017), [Link](https://doi.org/doi:10.1242/dev.143479).
35. K. Ishigaki et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat. Genet__. 49, 112
36. K. Tanegashima et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes to Cells__. 22, 71–83 (2017), [Link](https://doi.org/doi:10.1111/gtc.12456).
37. I. Yevshin, R. Sharipov, T. Valeev, A. Kel, F. Kolpakov, GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res__. 45, D61–D67 (2017), [Link](https://doi.org/doi:10.1093/nar/gkw951).
38. S. Baldi et al., Genome-wide rules of nucleosome phasing. __bioRxiv__, 093666 (2018), [Link](https://doi.org/doi:10.1101/093666).
39. N. Sugeno et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci. Rep__. 6, 36328 (2016), [Link](https://doi.org/doi:10.1038/srep36328).
