## To cite ChIP-Atlas in your publication:

- Preprint
  - Shinya Oki, Tazro Ohta, et al. Integrative analysis of transcription factor occupancy at enhancers and disease risk loci in noncoding genomic regions. bioRxiv 262899; doi: https://doi.org/10.1101/262899
- Database
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://dx.doi.org/10.18908/lsdba.nbdc01558-000
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://chip-atlas.org

---

## Publications citing ChIP-Atlas:

1. H. Kawaji, dirHub: a trackHub configurator with directory structure projection. __bioRxiv__, 314807 (2018), [Link](https://doi.org/doi:10.1101/314807).
2. A. Misawa, H. Orimo, lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif. Tissue Int.__ (2018), [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
3. T. Kehl et al., REGGAE: a novel approach for the identification of key transcriptional regulators. __Bioinformatics__ (2018), [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
4. L. M. Raffield et al., Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLOS Genet__. 14, e1007293 (2018), [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
5. K. Anan et al., LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res.__ (2018), [Link](https://doi.org/doi:10.1093/nar/gky234).
6. C. A. Lareau et al., Interrogation of human hematopoiesis at single-cell and single-variant resolution. __bioRxiv__, 255224 (2018), [Link](https://doi.org/doi:10.1101/255224).
7. E. Ferris, L. M. Abegglen, J. D. Schiffman, C. Gregg, Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep__. 22, 2742–2755 (2018), [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
8. S. Chatterjee et al., Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res. Treat.__ (2018), [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
9. I. Miura et al., DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J. Psychiatr. Res__. 100, 84–87 (2018), [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
10. F. Mao et al., EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res__. 46, D92–D99 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx918).
11. Y. Nishizawa et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__. 9 (2018), [Link](https://doi.org/doi:10.18632/oncotarget.23554).
12. T. Chishima, J. Iwakiri, M. Hamada, Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes (Basel)__. 9, 23 (2018), [Link](https://doi.org/doi:10.3390/genes9010023).
13. J. Chèneby, M. Gheorghe, M. Artufel, A. Mathelier, B. Ballester, ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res__. 46, D267–D275 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx1092).
14. R. Dréos, G. Ambrosini, R. Groux, R. C. Périer, P. Bucher, MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res__. 46, D175–D180 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx995).
15. Z. Kalender Atak et al., Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med__. 9, 80 (2017), [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
16. O. Cuvier, Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol__. 19, 1–15 (2017), [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
17. Y. Onodera et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__. 16, 1369–1380 (2017), [Link](https://doi.org/doi:10.1111/acel.12680).
18. T. Umeyama, T. Ito, DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep__. 21, 289–300 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
19. A. Naderi, C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__. 8, 57907–57933 (2017), [Link](https://doi.org/doi:10.18632/oncotarget.17826).
20. Y. Chen, M. Widschwendter, A. E. Teschendorff, Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol__. 18, 236 (2017), [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
21. T. Sanosaka et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep__. 20, 2992–3003 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
22. O. Govaere et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__. 36, 6605–6616 (2017), [Link](https://doi.org/doi:10.1038/onc.2017.260).
23. M. Yoshihara et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep__. 21, 308–315 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
24. F. K. Turrell et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev__. 31, 1339–1353 (2017), [Link](https://doi.org/doi:10.1101/gad.298463.117).
25. M. H. Guo et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc. Natl. Acad. Sci__. 114, E327–E336 (2017), [Link](https://doi.org/doi:10.1073/pnas.1619052114).
26. T. Kehl et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res__. 45, W146–W153 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx350).
27. K. Matsuda et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__. 144, 1948–1958 (2017), [Link](https://doi.org/doi:10.1242/dev.143479).
28. K. Ishigaki et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat. Genet__. 49, 1120–1125 (2017), [Link](https://doi.org/doi:10.1038/ng.3885).
29. K. Tanegashima et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes to Cells__. 22, 71–83 (2017), [Link](https://doi.org/doi:10.1111/gtc.12456).
30. I. Yevshin, R. Sharipov, T. Valeev, A. Kel, F. Kolpakov, GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res__. 45, D61–D67 (2017), [Link](https://doi.org/doi:10.1093/nar/gkw951).
31. N. Sugeno et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci. Rep__. 6, 36328 (2016), [Link](https://doi.org/doi:10.1038/srep36328).
