## To cite ChIP-Atlas in your publication:

- Preprint
  - Shinya Oki, Tazro Ohta, et al. Integrative analysis of transcription factor occupancy at enhancers and disease risk loci in noncoding genomic regions. bioRxiv 262899; doi: https://doi.org/10.1101/262899
- Database
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://dx.doi.org/10.18908/lsdba.nbdc01558-000
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://chip-atlas.org

---

## Publications citing ChIP-Atlas:

1. F. Mao et al., EpiDenovo: a platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res.__. 46, D92–D99 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx918).
2. T. Chishima, J. Iwakiri, M. Hamada, Identification of Transposable Elements Contributing to Tissue-Specific Expression of Long Non-Coding RNAs. __Genes (Basel).__. 9, 23 (2018), [Link](https://doi.org/doi:10.3390/genes9010023).
3. Y. Nishizawa et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__. 9 (2018), [Link](https://doi.org/doi:10.18632/oncotarget.23554).
4. A. Naderi, C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__. 8, 57907–57933 (2017), [Link](https://doi.org/doi:10.18632/oncotarget.17826).
5. K. Matsuda et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__. 144, 1948–1958 (2017), [Link](https://doi.org/doi:10.1242/dev.143479).
6. M. H. Guo et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc. Natl. Acad. Sci.__. 114, E327–E336 (2017), [Link](https://doi.org/doi:10.1073/pnas.1619052114).
7. T. Umeyama, T. Ito, DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep.__. 21, 289–300 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
8. T. Sanosaka et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep.__. 20, 2992–3003 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
9. K. Tanegashima et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes to Cells__. 22, 71–83 (2017), [Link](https://doi.org/doi:10.1111/gtc.12456).
10. I. Yevshin, R. Sharipov, T. Valeev, A. Kel, F. Kolpakov, GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res.__. 45, D61–D67 (2017), [Link](https://doi.org/doi:10.1093/nar/gkw951).
11. M. Yoshihara et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep.__. 21, 308–315 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
12. F. K. Turrell et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev.__. 31, 1339–1353 (2017), [Link](https://doi.org/doi:10.1101/gad.298463.117).
13. R. Dréos, G. Ambrosini, R. Groux, R. C. Périer, P. Bucher, MGA repository: a curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res.__, 1–6 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx995).
14. Y. Onodera et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__. 16, 1369–1380 (2017), [Link](https://doi.org/doi:10.1111/acel.12680).
15. K. Ishigaki et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat. Genet.__. 49, 1120–1125 (2017), [Link](https://doi.org/doi:10.1038/ng.3885).
16. T. Kehl et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res.__. 45, W146–W153 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx350).
17. J. Chèneby, M. Gheorghe, M. Artufel, A. Mathelier, B. Ballester, ReMap 2018: an updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res.__, 1–9 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx1092).
18. Y. Chen, M. Widschwendter, A. E. Teschendorff, Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol.__. 18, 236 (2017), [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
19. O. Govaere et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__ (2017), [Link](https://doi.org/doi:10.1038/onc.2017.260).
20. N. Sugeno et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci. Rep.__. 6, 36328 (2016), [Link](https://doi.org/doi:10.1038/srep36328).
